utilities/project_helpers/scripts/check-message-usages.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/python
+
+import sys
+import subprocess
+import codecs
+import os
+import re
+import xml.etree.ElementTree as xml
+
+from check_message_lib import find_language_file_name, root_directory
+
+language = sys.argv[1]
+
+line_regexp = r'^(.+?):(.*)$'
+
+def find_xml_prefixes_and_files():
+
+    prefixes = ['xmlui', 'homepage', 'input_forms', 'org.dspace', 'PiwikStatisticsTransformer', 'UFAL.firstpage']
+    grep_command = 'grep -R -P "[>\'\\"](' + '|'.join(prefixes) + ')\\." --include=*.java --include=*.xsl --include=*.xmap --include=*.xslt --include=input-forms.xml --exclude-dir=*/target/* *'
+    prefix_regexp = "[>'\"]((?:" + "|".join(prefixes) + ")\..+?)[<'\"]"
+
+    os.chdir(root_directory)
+    with open(os.devnull, 'w') as devnull:
+        output = subprocess.check_output(grep_command, shell=True, stderr=devnull)
```

Comment:
@cyplas getting the following error:
```
python utilities/project_helpers/scripts/check-message-usages.py cs
Traceback (most recent call last):
  File "utilities/project_helpers/scripts/check-message-usages.py", line 96, in <module>
    add_xml_results(language, results)
  File "utilities/project_helpers/scripts/check-message-usages.py", line 38, in add_xml_results
    message_prefixes = find_xml_prefixes_and_files()
  File "utilities/project_helpers/scripts/check-message-usages.py", line 24, in find_xml_prefixes_and_files
    output = subprocess.check_output(grep_command, shell=True, stderr=devnull)
AttributeError: 'module' object has no attribute 'check_output'
```
probably due to my ancient python 2.6.5. Is there anything I can do to run this with 2.6.5?